### DISCUSSION TITLE
"bulk" method from sql-template-tag?

### DISCUSSION THREAD

**Discussion by alexpmay** [prisma/prisma#21738] [Q&A] [answered]
### Question

Prisma reexports several sql-template-tag methods,

```
/**
 * Re-export of sql-template-tag
 */
export import sql = runtime.sqltag
export import empty = runtime.empty
export import join = runtime.join
export import raw = runtime.raw
export import Sql = runtime.Sql
```

But the [bulk](https://github.com/blakeembrey/sql-template-tag#bulk) method is not re-exported.

It would be super useful for examples like the one described in the sql-template-tag docs, when one has to fall back to `$queryRaw` or `$executeRaw`.

```
const query = `INSERT INTO users (name) VALUES ${bulk([
  ["Blake"],
  ["Bob"],
  ["Joe"],
])}`;

query.sql; //=> "INSERT INTO users (name) VALUES (?),(?),(?)"
query.values; //=> ["Blake", "Bob", "Joe"]
```

Is there a way to use `bulk` and/or any plans to add to Prisma?

Thank you,
Alex

**Comment by ludralph** [comment, marked as answer]
Hi @alexpmay 👋 

Thank you for raising this question.

Currently, there is no way to use `bulk` sql template tag. However, you can open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) and document your use case so that our team can take a look. 

Meanwhile, would this workaround documented in this previous [discussion](https://github.com/prisma/prisma/discussions/4177#discussioncomment-3937016) be relevant for you?

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by alexpmay** [comment]
Thank you @ludralph. I found that thread as well and ended up going with the approach listed in a [comment](https://github.com/prisma/prisma/discussions/4177#discussioncomment-3937016) there. [bulk](https://github.com/blakeembrey/sql-template-tag#bulk) would certainly be more straightforward, but the workaround works. I appreciate the prompt response, and I have marked your comment as the answer. Thanks again.
